zone/spell_effects.cpp

```diff
@@ -3565,7 +3565,7 @@ snare has both of them negative, yet their range should work the same:
 			if(ticdif < 0)
 				ticdif = 0;
 
-			result = updownsign * (ubase - (12 * ticdif));
+			result = ubase - (12 * ticdif);
```

Comment:
Removing this doesn't even seem to fix "Soul Well Recourse" it actually prevents the 1 method where it does work from working.
Without making this change I was able to get to "Soul Well Recourse" work by Setting it to the following:

![image](https://user-images.githubusercontent.com/109764533/210415259-c8f74fc3-ad27-4a55-a7fd-b341c3018b70.png)

Can see the Regen ticking up using #mystats (the client inventory though shows it broken)

![image](https://user-images.githubusercontent.com/109764533/210415080-ef2c6b6c-a46a-4d66-af4c-1939421f3fbc.png)

My actual recommendation is to ditch the Recourse on  "Soul Well" and implement it like Live has, it looks like they ditched the recourse aspect, and made it a resource tap.

![image](https://user-images.githubusercontent.com/109764533/210415699-245485af-736f-4910-ac25-1b4347c518ee.png)

![image](https://user-images.githubusercontent.com/109764533/210415668-9fa579c9-8bea-434f-a262-d01034b0a4c8.png)